api/CompressionStream.json

```diff
@@ -96,6 +100,11 @@
       },
       "readable": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CompressionStream/readable",
+          "spec_url": [
+            "https://wicg.github.io/compression/#compression-stream",
+            "https://streams.spec.whatwg.org/#dom-generictransformstream-readable"
```

Comment:
I'm not sure about this. Isn't this URL already on the corresponding entry in BCD? The main spec should be the entry point telling you everything you need to know about this member, otherwise I'd file a spec bug.

Reply:
On `iterable<T>` I think we should arrange for that declaration to be linkable in specs and link to it in each unique interface. I think it would be an issue for https://github.com/tabatkins/bikeshed/issues, if you agree do you want to file an issue?

But here I see what's going on with the mixins now. Linking both actually seems good to me, without the `includes` statement in https://wicg.github.io/compression/#compression-stream, it would be hard to understand why https://streams.spec.whatwg.org/#dom-generictransformstream-readable should be relevant, and it might like like an incorrect URL.

@Elchi3 have you dealt with this case before and arrived at a rule?

Reply:
For Iterable stuff like e.g. NodeList.entries we link to WebIDL only: https://github.com/mdn/browser-compat-data/blob/7f93ea53823df03a9af5c9505119388059c6b989/api/NodeList.json#L54

My feeling is that the generic link to https://wicg.github.io/compression/#compression-stream is not needed here.

Reply:
cc @sideshowbarker

Reply:
I don't think linking to https://heycam.github.io/webidl/#idl-iterable is that great, it's a bit like linking to https://heycam.github.io/webidl/#idl-attributes to explain why a property exists.

Maybe no spec link at all is fine until we can resolve https://github.com/mdn/browser-compat-data/issues/6367?

Reply:
> Maybe no spec link at all is fine until we can resolve #6367?

This seems like the best option to me. #6367 is the problem here basically.

Reply:
I propose the following: 
1. I update this PR following the convention used until now.
2. If we want to do differently in the future, we will have to update all of these (+toJSON +stringifier), so one more page will not change much.

Fix in next commit (just commiting something else first ;-) )

Reply:
Commit in.

Reply:
That sounds good to me! Can you also comment in https://github.com/mdn/browser-compat-data/issues/6367 what things one should search for to update all of this in the future?

Reply:
Comment added in #6367.